src/WingetCreateCLI/Commands/BaseCommand.cs

```diff
@@ -284,7 +284,15 @@ protected static async Task<string> DownloadPackageFile(string installerUrl)
 
                 if (e is HttpRequestException httpRequestException)
                 {
-                    Logger.ErrorLocalized(nameof(Resources.HttpResponseUnsuccessful_Error), httpRequestException.StatusCode);
+                    if (httpRequestException.HResult == -2147467259)
```

Comment:
This is just E_FAIL, which is a really generic error code. I'd instead check if StatusCode != null then display the original error message, otherwise just display ex.Message

Reply:
Changed to suggestion. Added check for StatusCode != null instead.